coffee_storage/src/nosql_db.rs

```diff
@@ -0,0 +1,49 @@
+use async_trait::async_trait;
+use nosql_db::NoSQL;
+use nosql_sled::SledDB;
+
+use coffee_lib::error;
+use coffee_lib::errors::CoffeeError;
+
+use crate::storage::StorageManager;
+
+/// No SQL database
+
+pub struct NoSQlStorage {
+    inner: SledDB,
+}
+
+impl NoSQlStorage {
+    pub fn new(path: &str) -> Result<Self, CoffeeError> {
+        let db = SledDB::new(path).map_err(|err| error!("{err}"))?;
```

Comment:
```suggestion
       // make the storage directory inside the `path`
        let db = SledDB::new(format!("{}/storage").map_err(|err| error!("{err}"))?;
```

ok I just forget that the database needs the full directory

Reply:
Done!